### PR TITLE
[MV-403] Better styling of the example app

### PR DESCRIPTION
**Pull request by skyman503**

### CODE REVIEW

File: app/src/main/java/com/dscout/membranevideoroomdemo/RoomActivity.kt [hunk 1]
```diff
@@ -121,50 +117,13 @@ class RoomActivity : AppCompatActivity() {
                                     viewModel.toggleVideoTrackEncoding(it)
                                 },
                                 colors = AppButtonColors(),
-                                modifier = Modifier.then(
```
Comment (graszka22) [inline app/src/main/java/com/dscout/membranevideoroomdemo/RoomActivity.kt:124]: why did you remove it? for video track we still use simulcast

File: app/src/main/java/com/dscout/membranevideoroomdemo/RoomActivity.kt [hunk 2]
```diff
@@ -221,28 +200,72 @@ fun ParticipantCard(
...
-            indication = null,
-            interactionSource = remember { MutableInteractionSource() }
-        ) {
-            onClick?.invoke()
+    fun shouldShowIcon(trackType: String): Boolean {
+        return when (trackType) {
+            "audio" -> {
+                participant.audioTrack == null || (
+                    participant.tracksMetadata.isNotEmpty() &&
+                        (participant.tracksMetadata[participant.audioTrack.id()]?.get("active") as? Boolean) != true
```
Comment (graszka22) [inline app/src/main/java/com/dscout/membranevideoroomdemo/RoomActivity.kt:208]: can we make it shorter or divide it into shorter statements? a three line check doesn't look too good

File: app/src/main/java/com/dscout/membranevideoroomdemo/viewmodels/RoomViewModel.kt [hunk 3]
```diff
@@ -147,6 +147,19 @@ class RoomViewModel(
             isMicrophoneOn.value = enabled
             room.value?.updateTrackMetadata(it.id(), mapOf("active" to enabled, "type" to "audio"))
         }
+        val p = mutableParticipants[localPeerId]
+        if (p != null) {
+            mutableParticipants[localPeerId] = p.copy(
+                tracksMetadata = p.tracksMetadata + (
```
Comment (graszka22) [inline app/src/main/java/com/dscout/membranevideoroomdemo/viewmodels/RoomViewModel.kt:153]: can we have this as a function that returns a new participant? see `Peer.withTrack` in the library. That way we'll reduce a loot of repetition.

**Review by graszka22 — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/dscout/membranevideoroomdemo/RoomActivity.kt (hunk 2 above)
Comment (skyman503) [reply app/src/main/java/com/dscout/membranevideoroomdemo/RoomActivity.kt]: added a new helper function

File: app/src/main/java/com/dscout/membranevideoroomdemo/viewmodels/RoomViewModel.kt (hunk 3 above)
Comment (skyman503) [reply app/src/main/java/com/dscout/membranevideoroomdemo/viewmodels/RoomViewModel.kt]: Done.

File: app/src/main/java/com/dscout/membranevideoroomdemo/viewmodels/RoomViewModel.kt [hunk 4]
```diff
@@ -372,31 +359,21 @@ class RoomViewModel(
...
-        if (ctx.metadata["type"] == "camera") {
-            if (p != null) {
-                mutableParticipants[ctx.peer.id] = p.copy(
-                    tracksMetadata = p.tracksMetadata + (
-                        (
-                            mutableParticipants[ctx.peer.id]?.videoTrack?.id()
-                                ?: ""
-                            ) to ctx.metadata
-                        )
+        if (mutableParticipants[ctx.peer.id] != null) {
```
Comment (graszka22) [inline app/src/main/java/com/dscout/membranevideoroomdemo/viewmodels/RoomViewModel.kt:363]: we can extract `mutableParticipants[ctx.peer.id]` as a new variable and that way it'll be shorter and we'll avoid `!!` operator

**Review by graszka22 — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/dscout/membranevideoroomdemo/viewmodels/RoomViewModel.kt (hunk 4 above)
Comment (skyman503) [reply app/src/main/java/com/dscout/membranevideoroomdemo/viewmodels/RoomViewModel.kt]: Done.

**Review by graszka22 — APPROVED** [review]
👍
